packages/api/app/Controllers/Http/InstitutionController.js

```diff
@@ -41,22 +48,55 @@ class InstitutionController {
 	 * POST /institutions
 	 */
 	async store({ request, response, auth }) {
-		const data = request.only(this.fields);
-		const institution = await Institution.create(data);
-		await institution.responsible().associate(auth.user);
+		const { logo_id, ...data } = request.only(this.fields);
+
+		let institution;
+		let trx;
+
+		try {
+			const { init, commit } = getTransaction();
+			trx = await init();
+			institution = await Institution.create(data, trx);
+			await institution.responsible().associate(auth.user, trx);
+			if (logo_id) {
+				const logo = await Upload.findOrFail(logo_id);
+				await institution.logo().associate(logo, trx);
+			} else {
+				institution.logi_id = null;
+			}
```

Comment:
Podemos retirar esse `else` e declarar null como o valor padrão do `logo_id`.
 ```suggestion
		const { logo_id = null, ...data } = request.only(this.fields);

		let institution;
		let trx;

		try {
			const { init, commit } = getTransaction();
			trx = await init();
			institution = await Institution.create(data, trx);
			await institution.responsible().associate(auth.user, trx);
			if (logo_id) {
				const logo = await Upload.findOrFail(logo_id);
				await institution.logo().associate(logo, trx);
			}
```